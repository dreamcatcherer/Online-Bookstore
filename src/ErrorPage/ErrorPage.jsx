import React from "react";
import pageNotFoundImage from "../images/404PageNotFound.jpg";
import styles from "./ErrorPage.module.css";

const PageNotFound = () => {
  return (
    <div className={styles.container}>
      <div>
        <img
          src={pageNotFoundImage}
          alt="404 Page Not Found"
          className={styles.image}
        />
      </div>
    </div>
  );
};
export default PageNotFound;