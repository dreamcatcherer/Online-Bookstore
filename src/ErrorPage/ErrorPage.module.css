.container {
  display: grid;
  grid-template-columns: 1fr;
}

.image {
  object-fit: cover;
  width: 100%;
  height: 100%;
  overflow: hidden;
}
