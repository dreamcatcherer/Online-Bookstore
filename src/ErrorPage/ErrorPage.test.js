import React from "react";
import { render } from "@testing-library/react";
import ErrorPage from "./ErrorPage.jsx";

it("renders the PageNotFound component correctly", () => {
  const { container } = render(<ErrorPage />);
  expect(container).toMatchSnapshot();
});
