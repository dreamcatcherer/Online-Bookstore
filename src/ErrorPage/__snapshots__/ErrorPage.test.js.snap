// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the PageNotFound component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <img
        alt="404 Page Not Found"
        class="image"
        src="404PageNotFound.jpg"
      />
    </div>
  </div>
</div>
`;
